cdhweb/static_src/global/layout/page-layout.scss

```diff
@@ -0,0 +1,62 @@
+// Has `.grid-standard`
+.page-layout {
+  & > * {
+    grid-column: 1 / -1;
+
+    @include md {
+      grid-column: 2 / -1;
+    }
+    @include lg {
+      grid-column: 3 / -2;
+    }
+  }
+}
+
+// From xl, with- and without-sidenav layouts diverge
+.page-layout--without-sidenav {
+  // m = main content
+  @include xl {
+    grid-template-areas: '. . m m m m m m m . . .';
+  }
+  @include xxxl {
+    grid-template-areas: '. . m m m m m m . . . .';
+  }
+}
+.page-layout--with-sidenav {
+  // m = main content
+  // s = side-nav
+
+  @include xl {
+    grid-template-areas: 's s s . m m m m m m m .';
+  }
+  @include xxxl {
+    grid-template-areas: 's s . m m m m m m . . .';
+  }
+}
+
+.page-layout__main-content {
+  @include xl {
+    grid-area: m;
+  }
+}
+
+.page-layout__sidenav {
+  // V space between sidenav and main content when stacked:
```

Comment:
V space?

Reply:
Vertical. No good? OK, will spell it out :)

Reply:
ah, yeah, spell it out please!